src/SOLID/SingleResponsabilityPrinciple/Success/BranchOfficeEmployeesManagement.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Deg540\PHPTestingBoilerplate\SOLID\SingleResponsabilityPrinciple\Success;
+
+class BranchOfficeEmployeesManagement
+{
+    private BranchOffice $branchOffice;
+    private array $employees;
+
+    public function __construct(BranchOffice $branchOffice)
+    {
+        $this->branchOffice = $branchOffice;
+        $this->employees = $this->branchOffice->getEmployees();
```

Comment:
Así nos podemos quitar la variable de BranchOffice. 
Si además, cambiamos el constructor a Employee[] nos quitaríamos conocer a BranchOffice. 
```suggestion
        $this->employees = $branchOffice->getEmployees();
```

Idem para BranchOfficeWasteCalculator